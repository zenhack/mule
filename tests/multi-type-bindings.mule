#XFAIL: type formatting
{
  , type t = int
  , type u = t
}
