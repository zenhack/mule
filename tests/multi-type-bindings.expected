
Desugared:

	(record (types (((t (rec t int))) ((u (rec u t))))) (values ()))

inferred type:

	(record (types ((u int) (t int))) (values ()))

Runtime term:

	(Record ())

Evaluated:

	(Record ())
